history/provinces/00_k_alterac.txt

```diff
@@ -31,6 +31,12 @@
 	culture = alteraci
 	religion = holy_light
 	holding = castle_holding
+	100.1.1 = {
+		special_building_slot = coldtooth_mine_01
+	}
+	120.1.1 = {
+		special_building = coldtooth_mine_01
+	}
```

Comment:
`Snivvle is here! Snivvle has claimed the Coldtooth Mine!`
we could have an event where a Kobold named Snivvle claims the mine. disables the building if you fail some kind of check. that would be fun!

Reply:
could be a one time event or an "as long as snivvle is alive" event

Reply:
He'll probably be a ruler when the map is overhauled and Alterac Valley is more than one county